vars/wum-pipeline.groovy

```diff
@@ -0,0 +1,101 @@
+import groovy.json.*
+
+pipeline {
+    agent any
+    environment {
+        //Set the jq path
+        PATH = "/usr/local/Cellar/jq/1.5_3/bin:$PATH"
```

Comment:
This is mac/homebrew specific folder. This won't work in linux. Btw, haven't we get rid of json parsing logic?

Reply:
removed the json logic and passing the text file to trigger jobs parallelly.